### PR TITLE
go-mockery: init at 2.5.1

### PR DESCRIPTION
**Pull request by cideM**
Also added myself as  `fbrs` to list of maintainers

<details>
<summary>Using the added package to verify it works</summary>

```shell
~/foo
$ nix-shell -p go gotools gopls
these 3 paths will be fetched (124.33 MiB download, 567.76 MiB unpacked):
  /nix/store/5cjd962ig8azhp45l7h95hl4jzkyzqv0-go-1.15.8
  /nix/store/fmc862521n4ssh2naklkz7agp2fjd8c4-gopls-0.4.4
  /nix/store/fyw7qcni6p6py0xlrxamydq1904hrbbg-gotools-unstable-2020-04-21
copying path '/nix/store/5cjd962ig8azhp45l7h95hl4jzkyzqv0-go-1.15.8' from 'https://cache.nixos.org'...
copying path '/nix/store/fmc862521n4ssh2naklkz7agp2fjd8c4-gopls-0.4.4' from 'https://cache.nixos.org'...
copying path '/nix/store/fyw7qcni6p6py0xlrxamydq1904hrbbg-gotools-unstable-2020-04-21' from 'https://cache.nixos.org'...

[nix-shell:~/foo]$ go mod init foo
go: creating new go.mod: module foo

[nix-shell:~/foo]$ nvim main.go

[nix-shell:~/foo]$ ~/nixpkgs/result/bin/mockery --name Foo --inpackage
03 Mar 21 16:07 CET INF Starting mockery dry-run=false version=0.0.0-dev
03 Mar 21 16:07 CET INF Walking dry-run=false version=0.0.0-dev
03 Mar 21 16:07 CET INF Generating mock dry-run=false interface=Foo qualified-name=foo version=0.0.0-dev

[nix-shell:~/foo]$ ls
go.mod  main.go  mock_Foo.go

[nix-shell:~/foo]$ cat mock_Foo.go
// Code generated by mockery v0.0.0-dev. DO NOT EDIT.

package main

import mock "github.com/stretchr/testify/mock"

// MockFoo is an autogenerated mock type for the Foo type
type MockFoo struct {
        mock.Mock
}

// Foo provides a mock function with given fields:
func (_m *MockFoo) Foo() string {
        ret := _m.Called()

        var r0 string
        if rf, ok := ret.Get(0).(func() string); ok {
                r0 = rf()
        } else {
                r0 = ret.Get(0).(string)
        }

        return r0
}
```
</details>

###### Motivation for this change

Package is a CLI tool for generating interface mocks and as such I think it's a good fit, in the same sense as `go-migrate` for example.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cideM** [comment]
Result of `nixpkgs-review pr 114985` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>go-mockery</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3071,6 +3071,12 @@
     githubId = 1276854;
     name = "Florian Peter";
   };
+  fbrs = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:3074]: Please put this into a separate commit with the message ``maintainers: add fbrs``.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (cideM) [reply maintainers/maintainer-list.nix:3074]: Done
